examples/performance/in-vr/components/stats-in-vr.js

```diff
@@ -0,0 +1,38 @@
+/* global AFRAME */
+AFRAME.registerComponent('stats-in-vr', {
+  schema: {
+    // Number of frames for moving average.
+    frames: {default: 30}
+  },
+
+  init: function () {
+    this.renderTimes = [];
+  },
+
+  tick: function (t, dt) {
+    var color;
+    var fps;
+    var renderTimes = this.renderTimes;
+
+    renderTimes.push(dt);
+    if (renderTimes.length > this.data.frames) { renderTimes.shift(); }
+
+    fps = renderTimes.length / (renderTimes.reduce(sum) / 1000);
+
+    color = 'green';
+    if (fps < 80) {
```

Comment:
Not sure if these ranges are useful. I think 8x is still an acceptable value since the asynchronous re projection of the different runtimes will fill the gaps. I bet that the stats will be most of the time `yellow` or `orange`. What about:
fps > 85 green
fps 70-85 yellow
fps 60-70 orange
fps > 60 red